tests/unit/peapods/runtimes/container/test_container_runtime.py

```diff
@@ -358,3 +283,52 @@ def test_pass_arbitrary_kwargs_from_yaml():
         'hello': 0,
         'environment': ['VAR1=BAR', 'VAR2=FOO'],
     }
+
+
+def test_container_override_params(docker_image_built, tmpdir, mocker):
+    def validate_response(resp):
+        assert len(resp.docs) > 0
+        for doc in resp.docs:
+            assert doc.tags['greetings'] == 'overriden greetings'
+
+    mock = mocker.Mock()
```

Comment:
not a big deal, but if you could also rewrite this with `return_results`, it would be sweet. anyway, I'll approve it